File-Formats/Presentation/Presentation-to-image.md

```diff
@@ -663,10 +667,156 @@ using (FileStream fileStreamInput = new FileStream(@"Template.pptx", FileMode.Op
 
 {% endtabs %}
 
-You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/PPTX-to-Image-conversion/Apply-custom-fallback-fonts).
+### Modify the exiting fallback fonts
+
+The following code example demonstrates how user can modify or customize the existing fallback fonts using *FontNames* API while converting a PowerPoint presentation to Image.
```

Comment:
The following code example demonstrates how users can modify or customize the existing fallback fonts using *FontNames* API while converting a PowerPoint presentation to an Image.

Reply:
Add a Comma at the end of each sentence.

Reply:
Modified the sentences